funcs/aws.go

```diff
@@ -65,6 +69,18 @@ func (a *Funcs) EC2Tag(tag string, def ...string) (string, error) {
 	return a.info.Tag(tag, def...)
 }
 
+// KMSEncrypt -
+func (a *Funcs) KMSEncrypt(keyID, plaintext string) (string, error) {
```

Comment:
args should be of type `interface{}` and should be converted inside the function (use `conv.ToString`)

Reply:
Sound good, thanks for the tip!